src/Driver/RedisProxySetDriver.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\HermesExtension\Driver;
+
+use Closure;
+use RedisProxy\RedisProxy;
+use RedisProxy\RedisProxyException;
+use Tomaj\Hermes\Dispatcher;
+use Tomaj\Hermes\Driver\DriverInterface;
+use Tomaj\Hermes\Driver\MaxItemsTrait;
+use Tomaj\Hermes\Driver\SerializerAwareTrait;
+use Tomaj\Hermes\Driver\ShutdownTrait;
+use Tomaj\Hermes\Driver\UnknownPriorityException;
+use Tomaj\Hermes\MessageInterface;
+use Tomaj\Hermes\MessageSerializer;
+use Tomaj\Hermes\Shutdown\ShutdownException;
+use Tomaj\Hermes\SerializeException;
+
+final class RedisProxySetDriver implements DriverInterface
+{
+    use MaxItemsTrait;
+    use ShutdownTrait;
+    use SerializerAwareTrait;
+
+    /** @var array<int, string>  */
+    private array $queues = [];
+
+    private RedisProxy $redis;
+
+    private int $refreshInterval;
+
+    public function __construct(RedisProxy $redis, string $key, int $refreshInterval = 1)
+    {
+        $this->setupPriorityQueue($key, Dispatcher::DEFAULT_PRIORITY);
+
+        $this->redis = $redis;
+        $this->refreshInterval = $refreshInterval;
+        $this->serializer = new MessageSerializer();
+    }
+
+    /**
+     * @throws RedisProxyException
+     * @throws SerializeException
+     * @throws UnknownPriorityException
+     */
+    public function send(MessageInterface $message, int $priority = Dispatcher::DEFAULT_PRIORITY): bool
+    {
+        $key = $this->getKey($priority);
+        return (bool)$this->redis->sadd($key, $this->serializer->serialize($message));
+    }
+
+    public function setupPriorityQueue(string $name, int $priority): void
+    {
+        $this->queues[$priority] = $name;
+    }
+
+    /**
+     * @throws RedisProxyException
```

Comment:
Maybe wrap all driver specific exceptions to under one driver exception that can be handled in dispatcher?

Reply:
there is nothing like catching Hermes exception in Dispatcher

```
} catch (Exception $exception) {
    if (Debugger::isEnabled()) {
        Debugger::log($exception, Debugger::EXCEPTION);
    }
}
```